ex/ThirdPartyDemos/martin-doyle/02 - Http Client Server ORM/source/Project02Server.dpr

```diff
@@ -0,0 +1,57 @@
+{:
+Synopse mORMot framework 2
+
+Sample 02 - Http Client Server ORM
+purpose of this sample is to show the basic ORM usage of the framework:
+}
+program Project02Server;
+
+{$APPTYPE CONSOLE}
+
+{$I mormot.defines.inc}
+{$define PUREMORMOT2}
```

Comment:
PUREMORMOT2 doesn't make much sense locally.
It should be enabled at the project level, in the options.

Reply:
Of course. Oops. Still some things to change for pure mORMot2.